release/RELEASE.md

```diff
@@ -1,14 +1,17 @@
 # Making a Release
 
-1. Make a PR that only bumps the version number in the VERSION
+1. Update https://github.com/digital-asset/daml/docs/support/release-notes.rst
```

Comment:
This sounds like it will require another CI roundtrip for every release which is something that I would like to avoid. Can we move this to the very end?

Reply:
Not if we want to release the docs together with the SDK at some point.

Reply:
Even if we don't want to release/publish at the same time, the version in the docs should still get updated. Otherwise we run into the same situation again that we have changes released into the SDK with the release-notes not being updated.

Reply:
Oh so this is about updating the number to the release that you are currently making. I understood this as adding a new header for changes after the one you are currently making. We should probably just adjust the release script to allow to do both at the same time.

Reply:
That'd be ideal. Can we merge this PR as is and do that in a second PR or would you like to help me do that as part of this PR?